src/components/CodeSnippet/CodeSnippet.js

```diff
@@ -7,7 +7,7 @@ const CodeSnippet = ({
   className,
   type,
   children,
-  onClick,
+  copyProps,
```

Comment:
Thanks @davidmenendez for improving the code to make allow consumers more control of copy button in code snippet! Looks that the change breaks the original `<CodeSnippet onClick={onClickHandler}>` case. Is it possible for you to update this PR to keep the backward-compatibility? Thanks!

Reply:
yeah for sure. it had occurred to me this would break it, but i thought my proposed solution was a good way to go just to have a clear separation between the two sets of props.

Reply:
@davidmenendez I see your solution is better, too - The consumers of this project has become sensitive for breaking changes, so we want to make sure we avoid breaking changes.